Fix grammar and language in write_test (#65)
taichi-dev/docs.taichi.graphics: website/docs/lang/articles/contribution/write_test.md

    assert r[None] == 2
```

## Using `ti.approx` for comparison with tolerance

> Sometimes the precision of maths operations could be poor on certain backends such as OpenGL,

Reviewer: ```suggestion
Sometimes the precision of math operations could be relatively low on certain backends such as OpenGL,
```

e.g. `ti.log10(100)` may return `2.001` or `1.999` in this case.

> To cope with this behavior, we provide `ti.approx` which can tolerate

Reviewer: ```suggestion
Adding tolerance with `ti.approx` can be helpful to mitigate
```

> tolerance won't vary among different Taichi backends. It'll be likely

Reviewer: line 125:

```diff
- Simply using `pytest.approx` won't work well here, since it's
+ Simply using `pytest.approx` won't work well here, since its
```

> tolerance won't vary among different Taichi backends. It'll be likely
> to fail on the OpenGL backend.

Reviewer: ```suggestion
tolerance won't vary among different Taichi backends. It'll likely
fail on the OpenGL backend.
```

`ti.approx` also correctly treats boolean types, e.g.:
`2 == ti.approx(True)`.
:::

## Parametrize test inputs

> In the test above, `r[None] = 100` means that it will only test that `ti.log10` works correctly for the input `100`. In order to test against different input values, we may use the `@pytest.mark.parametrize` decorator:

Reviewer: ```suggestion
In the test above, `r[None] = 100` means that it will only test that `ti.log10` works correctly for the input `100`. In order to test against different input values, you may use the `@pytest.mark.parametrize` decorator:
```

## Exclude some backends from test

> Sometimes some backends are not capable of certain tests, so we have to
> exclude them from the test:

Reviewer: ```suggestion
Some backends are not capable of executing certain tests, you may have to
exclude them from the test in order to move forward:
```

(Not 100% percent sure about this, but it feels to me that `sometimes` is somewhat redundant here...)
